code/_onclick/telekinesis.dm

```diff
@@ -12,27 +12,32 @@
 
 #define TK_MAXRANGE 15
 
-/atom/proc/attack_tk(mob/user)
-	if(user.stat || !tkMaxRangeCheck(user, src))
+/atom/proc/attack_tk(mob/user, is_weak = FALSE)
+	if(user.stat || !tkMaxRangeCheck(user, src) || is_weak)
 		return
 	new /obj/effect/temp_visual/telekinesis(get_turf(src))
 	user.UnarmedAttack(src,0) // attack_hand, attack_paw, etc
 	add_hiddenprint(user)
 	return
 
-/obj/attack_tk(mob/user)
+/obj/attack_tk(mob/user, is_weak = FALSE)
 	if(user.stat)
 		return
 	if(anchored)
 		return ..()
-	attack_tk_grab(user)
+	attack_tk_grab(user, is_weak)
 
-/obj/item/attack_tk(mob/user)
+/obj/item/attack_tk(mob/user, is_weak = FALSE)
 	if(user.stat)
 		return
-	attack_tk_grab(user)
+	attack_tk_grab(user, is_weak)
 
-/obj/proc/attack_tk_grab(mob/user)
+/obj/proc/attack_tk_grab(mob/user, is_weak = FALSE)
+	if(is_weak)
```

Comment:
This should be very specific override on a child

Reply:
Meant to completely remove this